src/isomorphic/classic/class/__tests__/ReactClass-test.js

```diff
@@ -355,7 +343,7 @@ describe('ReactClass-spec', function() {
     });
 
     expect(() => Component()).toThrow();
-    expect(console.error.calls.length).toBe(1);
+    expect(console.error.argsForCall.length).toBe(1);
```

Comment:
This one is fine either way - (it will actually be updated to `console.error.calls.count()` and argsForCall will become `console.error.calls.argsFor()` when I finally figure out the last bits I need for jasmine2). I know at one point we did change everything to use argsForCall consistently. I'll fix both so you do you :)